int32 robot_id
int32 stage
geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped goal
